mongofiles/mongofiles_test.go

```diff
@@ -137,6 +137,20 @@ func simpleMongoFilesInstanceWithFilenameAndID(command, fname, ID string) (*Mong
 	return &mongofiles, nil
 }
 
+// simpleMockMongoFilesInstanceWithFilename gets an instance of MongoFiles with no underlying SessionProvider.
+// Use this for tests that don't communicate with the server (e.g. options parsing tests)
+func simpleMockMongoFilesInstanceWithFilename(command, fname string) *MongoFiles {
+	return &MongoFiles{
+		ToolOptions:     toolOptions,
+		InputOptions:    &InputOptions{},
+		StorageOptions:  &StorageOptions{GridFSPrefix: "fs", DB: testDB},
+		SessionProvider: nil,
```

Comment:
nit: you don't need to initialize things to their zero-values. `SessionProvider` and `Id` can be skipped in the struct initialization.

Reply:
Fixed. Thanks for pointing this out (again). I'm used to swift where you need to explicitly define an initializer that allows you to omit certain properties.